### PR TITLE
Ecs web create test

### PR DESCRIPTION
**Pull request by aguilinger**
https://app.shortcut.com/aptible/story/7567/create-ecs-web

Same note as https://github.com/aptible/terraform-provider-aptible-iaas/pull/45 on roles and such since I used the same structure.

### CODE REVIEW

File: test/basic/ecs_web_create/ecs_web_create_test.go [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+	assert.Equal(t, *ecsServiceAws.Status, "ACTIVE")
+
+	ecsLoadBalancerUrl := terraform.Output(t, terraformOptions, "loadbalancer_url")
+	ecsLbGet, ecsLbGetErr := insecureHttpClient().Get(fmt.Sprintf("https://%s", ecsLoadBalancerUrl))
+	assert.Nil(t, ecsLbGetErr)
+	assert.EqualValues(t, ecsLbGet.StatusCode, 200)
+
+	ecsServiceUrl := terraform.Output(t, terraformOptions, "web_url")
+	ecsUrlGet, ecsUrlGetErr := http.Get(fmt.Sprintf("https://%s", ecsServiceUrl))
+	if assert.NoError(t, ecsUrlGetErr) {
```
Comment (madhuravius) [inline test/basic/ecs_web_create/ecs_web_create_test.go:145]: interesting, i always just do two assertions back to back 🤔 (`assert.Nil(t, errorthing`). TIL:

image

i usually use testify and/or goconvey and idk if they operate differently, but this lgtm

File: test/basic/ecs_web_create/ecs_web_create_test.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+			"ecs_name":          "ecs-web-test",
+			"container_command": []string{"nginx", "-g", "daemon off;"},
+			"container_image":   "nginx",
+			"container_port":    80,
+			"container_name":    "nginx",
+			"is_public":         true,
+			"is_ecr_image":      false,
+			"vpc_name":          "testecs-web-vpc",
+			"domain":            "aptible-cloud-staging.com",
+			"subdomain":         "test-ecs-integration",
```
Comment (madhuravius) [inline test/basic/ecs_web_create/ecs_web_create_test.go:104]: the only other var i see missing you might want to test (maybe in update command) is : 

```go
ContainerRegistrySecretArn types.String   `tfsdk:"container_registry_secret_arn"`
```

Comment (aguilinger) [reply test/basic/ecs_web_create/ecs_web_create_test.go]: I think that's a "flavor" of deployment that needs it's own tf and such. We can add that later when doing the "flavors" but for now I clarified which path this is

**Review by madhuravius — APPROVED** [review]
LGTM - if it works in tests

**Pull request merged**